app/src/main/res/layout/fragment_recipes.xml

```diff
@@ -4,17 +4,13 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="?attr/actionBarSize"
```

Comment:
This padding removal will cause Fragment UI to have a space at the top.
-Darren Chance
@leosamayoa

Reply:
Will fix on next commit to #10 
-Darren Chance